WireAnalytics/Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version: 5.10
+// swift-tools-version: 6.0
```

Comment:
🥳

Reply:
I am a bit lost in this changes, could you explain?

Reply:
This upgrades the WireAnalytics package to build with Swift version 6.